### DISCUSSION TITLE
clip raster brands

### DISCUSSION THREAD

**Discussion by hukjhnilh** [semiautomaticgit/SemiAutomaticClassificationPlugin#360] [General]
Hello, 
Hi, I have a problem with clipped rasters, I manage to clip them (I tried using coordinates and choosing the area manually), but I cannot work on them, after loading, the wrong center wavelenght is loaded in the brand set tab, which prevents further work, which may be wrong and how to fix it?
I'm attaching picture with situation.
![problem_clipped_raster](https://github.com/user-attachments/assets/f18f000f-f459-45f5-be37-a0c70ea63078)

**Comment by semiautomaticgit** [comment]
Hello @hukjhnilh , this is not a plugin issue.
After you clip the images you should create a new band set (or clear the active band set) and add the clipped bands. Then you should select the satellite name in "Band quick settings" to define the center wavelength of bands.

**Reply by hukjhnilh** [reply]
Okey, I got it but still it is problem with classification, look at photos
![450737082_1645819572884480_1935903272849589420_n](https://github.com/user-attachments/assets/1634a38a-3dbe-4f77-b3cc-890ca4998c6c)
![448761918_1406297506607060_1463158145102810701_n](https://github.com/user-attachments/assets/0f55b6be-43cd-40c9-8b5b-855979790424)
![450678666_833494255399055_2301974587316356337_n](https://github.com/user-attachments/assets/b3bccdd2-a302-4192-9ad3-6ed066ab520c)
![450488901_1035674294613817_796591120310734643_n](https://github.com/user-attachments/assets/73b662b8-8811-4d92-a79a-a54cbb76893a)

**Comment by semiautomaticgit** [comment]
Hello @hukjhnilh ,
from you settings you are using the Macroclass ID for classification, but you assigned the same value to all the ROIs (i.e. 1).
In your case you should set Use Class ID.
Please follow this tutorial https://semiautomaticclassificationmanual.readthedocs.io/en/latest/tutorial_1.html
